# The camera intrinsic parameters matrix
# 3x3 matrix
camera_intrinsic_parameters: [700,   0,    960,
                              0,    700,   540,
                              0,     0,     1]

# Sampling time
dt: 0.01

# Gain on linear velocities
gainv: 4

# Gain on angular velocities
gainw: .3

# Threshold on the feature position error, used to stop the visual control
feature_error_threshold: .01

nfeatures: 500
scaleFactor: 1.2
nlevels: 8
edgeThreshold: 15
patchSize: 30
fastThreshold: 20
flann_ratio: 0.7

############################################################
# Cambio de control 1 (1/dist) o 2 (dist).
control: 1

############################################################
# Cambio de c치mara 0, 1 o 2.
# 1 Para c치mara de iris - frontal
# 2 Para c치mara de iris - abajo
# 3 Para c치mara de iris - Ambas
camara: 1